README.md

```diff
@@ -1 +1,25 @@
-# spring-ecommerce-product
\ No newline at end of file
+# spring-ecommerce-product
+
+## Product
+  -[x] Create a simple CRUD HTTP API
+- [x] Create a Product
```

Comment:
<img width="442" height="259" alt="Screenshot 2025-07-19 at 10 10 37 PM" src="https://github.com/user-attachments/assets/efdf29d4-c418-41f6-9838-a0ba1b08c8ce" />

There seems to be an extra space here!